src/people.rs

```diff
@@ -174,7 +267,7 @@ pub trait ContextPeopleExt {
     /// initializing it if it hasn't been set yet. If no initializer is
     /// provided, and the property is not set this will panic
     fn get_person_property<T: PersonProperty + 'static>(
-        &self,
+        &mut self,
```

Comment:
Having `get_person_property` require a mutable reference is probably not what we want – that would be a pretty big changed (since we want to provide read access to properties in scenarios where you only have an immutable reference to context)